.github/CODEOWNERS

```diff
@@ -64,10 +64,10 @@ exporter/sentryexporter/                             @open-telemetry/collector-c
 exporter/signalfxexporter/                           @open-telemetry/collector-contrib-approvers @pmcollins @dmitryax
 exporter/skywalkingexporter/                         @open-telemetry/collector-contrib-approvers @liqiangz
 exporter/splunkhecexporter/                          @open-telemetry/collector-contrib-approvers @atoulme @dmitryax
-exporter/sumologicexporter/                          @open-telemetry/collector-contrib-approvers @pmm-sumo @sumo-drosiek
+exporter/sumologicexporter/                          @open-telemetry/collector-contrib-approvers @swiatekm-sumo @sumo-drosiek
```

Comment:
I support removing @pmm-sumo, but we've decided that all [code owners going forward need to be members of the OpenTelemetry organization](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner), and there was not an exception made for vender-specific components.  Is there anyone else from Sumo that is already a member that could be added?

Reply:
@swiatekm-sumo considering that you [already made several contributions](https://github.com/open-telemetry/opentelemetry-collector-contrib/pulls?q=is%3Apr+author%3Aswiatekm-sumo+is%3Aclosed) I would suggest to apply for membership as the preferred solution :)

Reply:
@TylerHelmuth I haven't seen followup on my suggestion, so to avoid blocking this PR further, I removed this entry. I believe this was the only outstanding item